terragrunt/aws/api/sns.tf

```
@@ -0,0 +1,12 @@
+#
+# SNS: topic & subscription
+#
+resource "aws_sns_topic" "lambda_dead_letter" {
+  name = "cloudwatch-alarms-lambda-dead-letter"
```

Could you change the name of this topic to something like:
```suggestion
  name = "${var.product_name}-api-dead-letter-queue"
```